crypto/bio/b_sock2.c

```diff
@@ -115,6 +115,57 @@ int BIO_connect(int sock, const BIO_ADDR *addr, int options)
     return 1;
 }
 
+/*-
+ * BIO_localaddr - Sets local address for socket
+ * @sock: the socket to set
+ * @addr: local address to bind to
+ * @options: BIO socket options
+ *
+ * Binds to the address using the given socket and options, then
+ * starts listening for incoming connections.
```

Comment:
No it doesn't, it doesn't start listening that is.

Reply:
Lazy copy'n'paste error on my part